Sources/Playground.swift

```diff
@@ -307,7 +307,7 @@ func displayHelp() {
                 Default: Any code specified with -c or its default value
         💪  -f  Force overwrite any existing playground at the target path
                 Default: Don't overwrite, and instead open any existing playground
-        ℹ️  -h  Display this information
+        ℹ️   -h  Display this information
```

Comment:
This shouldn't be here right? As it will cause the indentation to be different for `-h` compared to the other parameters.

Reply:
Interesting, for me it had the opposite effect - the additional space was necessary for correct alignment. Maybe something to do with my German locale?

Reply:
Maybe, or the font that you use in your terminal? 🤔